src/dotnet/Chat.UI.Blazor/Components/Banners/TalkingMembersBanner.razor

```diff
@@ -0,0 +1,88 @@
+@namespace ActualChat.Chat.UI.Blazor.Components
+@inherits ComputedStateComponent<TalkingMembersBanner.Model>
+@{
+    var m = State.Value;
+    if (ReferenceEquals(m, Model.None))
+        return;
+
+    if (!ScreenSize.IsNarrow())
+        return;
+}
+
+<Banner IsVisible="@m.IsVisible" Severity="BannerSeverity.Default" Class="talking-members-banner subheader">
+    <Icon>
+        <i class="icon-voice-01 text-danger text-2xl"></i>
+    </Icon>
+    <Body>
+    <div class="c-activities-container">
+        <ChatActivities @key="@Chat.Id.Value" Chat="@Chat" MaxIconCount="2"/>
+    </div>
+    </Body>
+    <Buttons>
+        @if (!IsListening) {
+            <Button Class="btn-primary talking-banner-btn" Click="@OnListenClick">
+                <Icon><i class="icon-listen text-xl"></i></Icon>
+                <Title>Listen</Title>
+            </Button>
+        }
+    </Buttons>
+</Banner>
+
+@code {
+    private Session Session => ChatContext.Hub.Session();
```

Comment:
Extract Hub property to make accessing services shorter:
private ChatUIHub Hub => ChatContext.Hub;
private AccountUI AccountUI => Hub.AccountUI;